react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -61,10 +61,11 @@
   opacity: 0.5;
 }
 
-.subItem {
-  @media @desktop {
-    background-color: @sk-gray-light;
-  }
+.menuSeparator {
+  height: 1px;
+  border: 0;
+  background-color: @sk-mid-gray-light;
+  margin: @row-height @row-height (@row-height - 1);
```

Comment:
Two minor points here:
1. I suggest extracting the `height` value to something like `@separator-height` and using it in this calculation to make the code more self documenting.
2. The horizontal margin dimension should not be a factor of rows. Typically we use a variation of the `@gutter-width`, eg:
```less
margin: @row-height (@gutter-width / 2) @separator-height
```